robot/Cumulus/tests/README.md

```diff
@@ -0,0 +1,9 @@
+## NPSP Robot test tags
+
```

Comment:
Love the readme!

I did notice some other tags that might be useful to add here if they have special meaning:
* notonfeaturebranch
* feature:GAU

I suppose it's possible the `feature` ones were added just as way of categorizing them by what's being tested in which case we probably don't need to list them all out. The ocd part of me wants to change all the occurrences of `feature:crlps` to `feature:crlp`. That'll wait for another day. 😄 

Also, do you think the work item numbers in the tests still have meaning/value?

Reply:
@force2b Those are great points.  

I'll add the missing tags to the readme and fix the feature:crlp spelling now - seems like as good a time as any.  

I'll also remove all the work item tags.  I don't believe they serve any purpose and they clutter things up - better to remove them.